.packit.yaml

```diff
@@ -84,3 +84,173 @@ jobs:
     post-upstream-clone:
     # builds from master branch should start with 100 release, to have high priority
     - bash -c "sed -i \"s/1%{?dist}/100%{?dist}/g\" packaging/leapp-repository.spec"
+
+- job: tests
+  fmf_url: "https://gitlab.cee.redhat.com/oamg/tmt-plans"
+  fmf_ref: "master"
+  use_internal_tf: True
+  trigger: pull_request
+  targets:
+    epel-7-x86_64:
+      distros: [RHEL-7.9-ZStream]
+  identifier: tests-7.9to8.6
+  tmt_plan: "^(?!.*c2r)(?!.*sap)(?!.*8to9)"
```

Comment:
Should be `^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*8to9)(?!.*max_sst)`

You can use https://github.com/oamg/leapp-repository/blob/master/.github/workflows/tmt-tests.yml for reference

Reply:
done.